packages/graphback/src/GraphbackRuntime.ts

```diff
@@ -26,6 +27,18 @@ export class GraphbackRuntime {
       config: { crudMethods: this.config.crud }
     });
     this.metadata = pluginEngine.createSchema();
+
+    const models = this.metadata.getModelDefinitions();
+    this.modelConfigs = models.map((model: ModelDefinition) => {
```

Comment:
Note: This was hack to keep similar code between codegen and runtime.

When moving to runtime we do not need that config at all as it duplicates .graphqlrc.yml.config. In codegen we did not needed that.